src/App.js

```diff
@@ -19,6 +21,7 @@ export default function App() {
   }
 
   const getTaskColumns = task => {
+    return
```

Comment:
return prematurely because this code now breaks... might need to be refactored after the data is combined